### PR TITLE
Prac 06 code review request

### PR DESCRIPTION
**Pull request by BrockAllen-au**
Hi @Shad-Man99, please review my prac_06. Any feedback welcome :-)

### CODE REVIEW

File: prac_06/used_cars.py [hunk 1]
```diff
@@ -9,10 +9,19 @@
...
     print(my_car)
+    # 1 Create a new Car object called limo
+    limo = Car("Limo", 100)
+    # 2 Add 20 units of fuel using the add method
+    limo.add_fuel(20)
+    # 3 Print amount of fuel in the car
+    print(f"Limo has fuel: {limo.fuel}")
+    # 4 Drive the car 115km using the drive method
+    limo.drive(115)
+    print(limo)
```
Comment (Shad-Man99) [inline prac_06/used_cars.py:25]: Maybe add spaces after each section, looks a bit congested.

File: prac_06/guitar.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+"""
+CP1404/CP5632 Practical - Guitar Class.
+Expected time to write class: 20mins
+Start time: 6:30pm
+"""
```
Comment (Shad-Man99) [inline prac_06/guitar.py:6]: Is there more to this?

File: prac_06/programming_language.py [hunk 3]
```diff
@@ -1 +1,21 @@
 """CP1404/CP5632 Practical - ProgrammingLanguage Class."""
+
```
Comment (Shad-Man99) [inline prac_06/programming_language.py:2]: Looks good, nice use of variables.

File: prac_06/guitar.py [hunk 4]
```diff
@@ -3,3 +3,27 @@
 Expected time to write class: 20mins
 Start time: 6:30pm
 """
+CURRENT_YEAR = 2022
+
```
Comment (Shad-Man99) [inline prac_06/guitar.py:8]: Forgot to add a timestamp.

File: prac_06/languages.py [hunk 5]
```diff
@@ -0,0 +1,5 @@
+"""
+CP1404/CP5632 Practical - Client code to use the ProgrammingLanguage class.
+Estimated completion time: 20mins
+Start Time: 6:00pm
+"""
```
Comment (Shad-Man99) [inline prac_06/languages.py]: Looks incomplete

**Review by Shad-Man99 — COMMENTED** [review]
Just a few things to fix but overall good work.

File: prac_06/used_cars.py (hunk 1 above)
Comment (Shad-Man99) [inline prac_06/used_cars.py:25]: Looks good, nice use of docstring comments.

File: prac_06/guitars.py [hunk 6]
```diff
@@ -2,6 +2,29 @@
...
+
+
+def main():
+    """My Guitars program"""
+    guitars = []
+    print("My guitars!")
+    guitar_name = input("Name: ")
+    while guitar_name != "":
+        year = int(input("Year: "))
+        cost = float(input("Cost: $"))
```
Comment (Shad-Man99) [inline prac_06/guitars.py:17]: Could have made cost to 2 decimal places.

**Comment by Shad-Man99** [comment]
Hey man,

Could you please confirm you received my code review? (i.e. me reviewing your code)



________________________________

**Comment by BrockAllen-au** [comment]
Thanks @Shad-Man99, review received.

**Comment by Shad-Man99** [comment]
Cheers 🙂
________________________________

**Pull request merged**